go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/Masterminds/semver v1.5.0 // indirect
 	github.com/Masterminds/sprig v2.22.0+incompatible // indirect
 	github.com/arschles/assert v1.0.0
+	github.com/davecgh/go-spew v1.1.1
```

Comment:
This one slipped through the cracks :slightly_frowning_face:. Needed it for debugging.

Reply:
you should be able to remove it now